cmd/server/main.go

```diff
@@ -42,7 +42,12 @@ import (
 // main entry point for the web server
 func main() {
 	app := buildCLI()
-	_ = app.Run(os.Args)
+	err := app.Run(os.Args)
+	if err != nil {
+		// An unhandled error was returned, wrap it and run it through the default exit code handler. Any errors
+		// that make it here should be caught further up the call stack and wrapped with cli.Exit and the proper exit code.
+		cli.HandleExitCoder(cli.Exit(fmt.Sprintf("Unexpected error encountered: %v.", err), 9))
```

Comment:
would rather leave just the error w/o the additional `Unexpected error..`

Reply:
is the addition to find errors easier in the logs?

Reply:
the idea was that the errors should never make it to this point, they should be handled at the Action level.

this only logs if an error was returned without `cli.Exit`.